src/components/Counter.test.js

```diff
@@ -0,0 +1,59 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom'
+import Counter from "./Counter";
+import { act } from 'react-dom/test-utils';
+
+describe("Check Counter element behavior", () => {
+
+    test("Test that component renders initial value provided in props", () => {
+
+        const propValue = 2;
+
+        render(<Counter defaultValue={propValue}/>)
+
+        const propValueElement = screen.getByText(/count/i);
```

Comment:
Note: Good practice is to check the letter's case as well. It makes a test case more reliable.